src/store/reducers/belReducer.js

```diff
@@ -0,0 +1,56 @@
+import { RETURN_EXCHANGE_DETAILS } from '../actionTypes'
+
+const initialState = {
+    quantityToReturn: 0,
+    quantityToExchange: 0,
+    keepRefundExchange: null,
+    refundReason: null,
+    exchangeReason: null,
+    exchangeToSize: null,
+    itemComments: null
+}
+
+
+export default function(state = initialState, action) {
+    switch(action.type) {
+        case RETURN_EXCHANGE_DETAILS:
+     //     window.alert(action.payload.quantityToReturn);
+            const { quantityToReturn, quantityToExchange, keepRefundExchange, refundReason, exchangeReason, exchangeToSize, itemComments } = action.payload
+            return {
+                ...state,
+                quantityToReturn,
+                quantityToExchange,
+                keepRefundExchange,
+                refundReason,
+                exchangeReason,
+                exchangeToSize,
+                itemComments
+            }
+
+        default:
+
```

Comment:
Default reducer handler should return the existing state.

```suggestion
        default:
            return {...state}
```